src/app/components/options-sidebar/time-clock/time-clock.component.ts

```diff
@@ -72,15 +74,20 @@ export class TimeClockComponent  implements OnInit {
    }
 
    setShowFields(show: boolean) {
-    this.isClockIn = false;
-    this.showFields = show;
-    this.startTimer();
-    this.setTimeToInOut();
+     if ( this.isClockInEnable !== true ) {
+      this.isClockIn = false;
+      this.showFields = show;
+      if (  this.execOnlyOneTimeCounter === 0 ) {
```

Comment:
why ===0?
What is going to happen if not?
I'd rather do something like:
```
if(!isTimerStarted){
   this.startTimer();
   isTimerStarted = true;
}
```

Reply:
done